### DISCUSSION TITLE
Are there any limitations to what resources we can access in a slash command extension?

### DISCUSSION THREAD

**Discussion by nicholasq** [zed-industries/zed#18319] [Feature Requests]
I am trying to create a slash command for myself that executes a shell command and I get this error:
`error: Error executing murl: Failed to execute murl: operation not supported on this platform`

I also tried making another extension that makes an http call to a local server on the machine and I get the same type of error above. So it had me wondering if there are restrictions to what a slash command is allowed to do?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
